org.DolphinEmu.dolphin-emu.json

```diff
@@ -83,6 +86,16 @@
         {
           "type": "file",
           "path": "appdata.xml"
+        },
+        {
+          "type": "script",
+          "commands": [
+            "for i in {0..9}; do",
```

Comment:
Only `0` is ever used AFAIK?

Reply:
The docs says that
>Discord can be on any pipe ranging from discord-ipc-0 to discord-ipc-9

but we a exporting only `0` anyway.

Reply:
I copied that snippet from https://github.com/flathub/com.discordapp.Discord/wiki/Rich-Precense-(discord-rpc), should I create only ``discord-ipc-0``?

Reply:
IMO better leave it as is, redundant symlinks inside the sandbox are harmless.